themes/finna2/js/finna-layout.js

```diff
@@ -381,6 +381,10 @@ finna.layout = (function finnaLayout() {
           holder.addClass('opened');
           VuFind.itemStatuses.check(holder);
           finna.itemStatus.initDedupRecordSelection(holder);
+          setTimeout(function onTriggerUnveil () {
+            holder.find('.recordcover').trigger('unveil');
+          }, 150);
```

Comment:
Vähän huolettaa luottaminen siihen, että timeoutin odottaminen riittää. Olisiko parempi tehdä niin, että siinä tapauksessa, että holderilla ei ole opened-luokkaa, tehtäisiin `holder.find('.recordcover').trigger('unveil');` slideToggle:n complete-handlerissa? Vaatii vähän säätöä järjestykseen, jossa jutut tehdään, mutta olisi varmatoiminen.

Reply:
Joo, paljon parempi näin.